mssql-plugin/src/e2e-test/features/mssql/mssql source/RunTimeWithMacros.feature

```diff
@@ -115,3 +115,169 @@ Feature: Mssql Source - Run time scenarios (macro)
+    And Enter runtime argument value from environment variable "password" for key "Password"
+    And Run the Pipeline in Runtime with runtime arguments
+    Then Wait till pipeline is in running state
+    Then Open and capture logs
+    Then Verify the pipeline status is "Succeeded"
+
+  @MSSQL_SOURCE_TEST @MSSQL_SINK_TEST @Mssql_Required
+  Scenario: To verify data is getting transferred from Mssql to BigQuery successfully using macros for Connection section
+    Given Open Datafusion Project to configure pipeline
+    When Expand Plugin group in the LHS plugins list: "Source"
+    When Select plugin: "SQL Server" from the plugins list as: "Source"
+    When Expand Plugin group in the LHS plugins list: "Sink"
+    When Select plugin: "BigQuery" from the plugins list as: "Sink"
+    Then Connect plugins: "SQL Server" and "BigQuery" to establish connection
+    Then Navigate to the properties page of plugin: "SQL Server"
+    Then Select dropdown plugin property: "select-jdbcPluginName" with option value: "driverName"
+    Then Replace input plugin property: "host" with value: "host" for Credentials and Authorization related fields
+    Then Replace input plugin property: "port" with value: "port" for Credentials and Authorization related fields
+    Then Replace input plugin property: "user" with value: "username" for Credentials and Authorization related fields
+    Then Replace input plugin property: "password" with value: "password" for Credentials and Authorization related fields
+    Then Enter input plugin property: "referenceName" with value: "sourceRef"
+    And Click on the Macro button of Property: "database" and set the value to: "DatabaseName"
+    And Click on the Macro button of Property: "fetchSize" and set the value to: "fetchSize"
+    Then Enter textarea plugin property: "importQuery" with value: "selectQuery"
+    Then Validate "SQL Server" plugin properties
+    Then Close the Plugin Properties page
+    Then Save the pipeline
+    Then Preview and run the pipeline
+    Then Enter runtime argument value "databaseName" for key "DatabaseName"
+    Then Enter runtime argument value "fetchSize" for key "fetchSize"
+    Then Verify the preview of pipeline is "success"
+    Then Close the preview
+    Then Deploy the pipeline
+    Then Run the Pipeline in Runtime
+    Then Enter runtime argument value "databaseName" for key "DatabaseName"
+    Then Enter runtime argument value "fetchSize" for key "fetchSize"
+    And Run the Pipeline in Runtime with runtime arguments
+    Then Wait till pipeline is in running state
+    Then Open and capture logs
+    Then Verify the pipeline status is "Succeeded"
+
+    @MSSQL_SOURCE_TEST @MSSQL_SINK_TEST @Mssql_Required
+    Scenario: Verify pipeline failure message in logs when user provides invalid Table name in importQuery of plugin with MacrosScenario: To verify data is getting transferred from Mssql to Mssql successfully
```

Comment:
Same comment here we are not validating the failure message in logs. There is a step to verify logs message use that step in all the scenarios where you are validating error message.

Reply:
resolved